frontend/src/components/Pages/MissionDefinitionPage/MissionDefinitionPage.tsx

```diff
@@ -48,91 +59,138 @@ const StyledMissionDefinitionPage = styled.div`
     gap: 1rem;
     margin: 2rem;
 `
+const StyledButton = styled(Button)`
+    width: 150px;
+`
+const StyledInspectionFrequencyDiv = styled.div`
+    > * {
+        padding: 10px;
+    }
+`
+const StyledEditButton = styled(Button)`
+    padding-left: 5px;
+    padding-top: 0px;
+    margin-top: 0px;
+    height: 0px;
+`
 
-interface IProps {
-    missionDefinition: CondensedMissionDefinition
-    updateMissionDefinition: (missionDefinition: CondensedMissionDefinition) => void
-}
-
-function KeyValuePairDisplay({ left, right }: { left: string; right: any }) {
+function MetadataItem({ title, content, onEdit }: { title: string; content: any; onEdit?: () => void }) {
     return (
-        <>
+        <StyledFormItem>
             <Typography
                 variant="body_long_bold"
                 group="paragraph"
                 color={tokens.colors.text.static_icons__secondary.rgba}
             >
-                {left}
+                {title}
+                {onEdit && (
+                    <StyledEditButton variant="ghost" onClick={onEdit}>
+                        <Icon name={Icons.Edit} size={16} />
```

Comment:
![image](https://github.com/equinor/flotilla/assets/33518988/2bdf91cf-14af-48c6-a859-552b94aded53)

The edit icon looks a bit weird when hovering

Reply:
Would you prefer if I disabled onHover for it?